distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/restito/ChunkedContent.java

```diff
@@ -298,8 +343,11 @@ private Response send(Response response, ByteRange byteRange) {
                     } else {
                         end = Integer.parseInt(endToken);
                     }
-                }catch (NumberFormatException e){
-                    LOGGER.error("Incoming request's range header is improperly formatted: [range={}]", rangeHeaderValue, e);
+                } catch (NumberFormatException e) {
+                    LOGGER.error(
+                            "Incoming request's range header is improperly formatted: [range={}]",
+                            rangeHeaderValue,
+                            e);
```

Comment:
I'm trying to think my way through the `isInfinite` loop. Tell me if I have it right or not?

If the chunk is infinitely big, then
  1. Write one character to the response buffer
  2. Flush the buffer
  3. Wait
  4. If the number of actual failures is less than the number of desired failures, 
      Then
          a. Increment the actual number of failures (counter)
          b. Recycle the buffer and return response. This simulates a dropped connection.
     Else
          Go back to step #1.

As long as the `Action` object exists, the state of the `ChunkedContentFunction` exists. Subsequent calls to `ChunkedContentFunction.send()` will write a single character, then recycle buffer and return the response because ` if (numberOfRetries < numberOfFailures)` will be already be true.

Do I have it right?